db/migrate/20170327104900_create_resturants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateResturants < ActiveRecord::Migration[5.0]
+  def change
+    create_table :resturants do |t|
```

Comment:
I'd add database level validations on top of the model-level ones (like can't be null, needs to be unique etc.)